src/core_atmosphere/physics/mpas_atmphys_driver_seaice.F

```diff
@@ -504,21 +509,24 @@ subroutine seaice_to_MPAS(configs,mesh,diag_physics,sfc_input,its,ite)
  character(len=StrKIND),pointer:: sfclayer_scheme
 
  real(kind=RKIND),dimension(:),pointer:: acsnom,acsnow,chs,chs2,cpm,cqs2,qgh,qsfc,grdflx,hfx, qfx,lh,noahres, &
-                                         potevp,sfc_albedo,sfc_emiss,sfcrunoff,snopcx,z0,znt
+                                         potevp,sfc_albedo,sfc_albbck,sfc_emiss,sfcrunoff,snopcx,z0,znt
  real(kind=RKIND),dimension(:),pointer:: snow,snowc,snowh,skintemp,xice
  real(kind=RKIND),dimension(:,:),pointer:: tslb
 
- real(kind=RKIND),dimension(:),pointer  :: cqs,dew,qcg,qsg,qvg,snowfallac,soilt1,tsnav
+ real(kind=RKIND),dimension(:),pointer  :: cqs,flqc,flhc,dew,qcg,qsg,qvg,snowfallac,soilt1,tsnav
 
  integer,pointer:: globalCells(:)
 
 !local variables and arrays:
  integer:: i,j,n
+!local pointers:
+ logical,pointer:: config_frac_seaice
```

Comment:
from a picky organizational standpoint, I would put this line with the other config pointers (between line 508 and 509)

Reply:
the change is made